.github/workflows/build_and_test_gcc_debug.yml

```diff
@@ -1,5 +1,7 @@
 name: Debug build (gcc)
-on: [push, pull_request]
+on:
```

Comment:
CI for this PR seems to be waiting for gcc builds CONCORD_BFT_CONTAINER_CC=gcc CONCORD_BFT_CONTAINER_CXX=g++. So wouldn't disabling this to weekly mean that it should not have run until saturday and must not wait but pass post clang runs?

Reply:
Hi Celine & Teo,
I understand that without completing gcc CI jobs for both release and debug I cannot merge this PR, also it effects other PR's too.

Is there anything I can configure from my side to discard gcc CI jobs like asan and tsan which will be trigger on schedule time in background without affecting other PR's requests ?

Thanks.

Reply:
Hi Akash, I don't know the answer to that. Please research and come up with a suggestion.

Reply:
Let me take a look if I find something.

Thanks

Reply:
One option would be to wait till Saturday for the jobs to be triggered, but it doesn't sound optimal... Maybe @KTeofanidis can give us a hint how to work around this?

Reply:
I found the solution but it requires admin credentials for the concord-bft repo where these branch protection rules are created.
So ultimatly we have to update branch protection rules for vmware/concord-bft to work for all pull request and merging those. I need someone who can help me with admin credentials.

Thanks

Reply:
Thanks Akash, please contact us Yulia Sherman who I believe has admin rights.

Reply:
Thank Teo for updating. I will connect with Yulia.

Reply:
Hi Teo,
So its done. Can I proceed with merging this pull request?